ci/datamgr.py

```diff
@@ -207,11 +207,31 @@ def postgres(schema, tables, data_directory, psql_path, **params):
     engine = init_database(
         'postgresql', params, schema, isolation_level='AUTOCOMMIT'
     )
+    if 'geo' in tables:
+        engine.execute("CREATE EXTENSION POSTGIS")
 
     query = "COPY {} FROM STDIN WITH (FORMAT CSV, HEADER TRUE, DELIMITER ',')"
     database = params['database']
     for table in tables:
         src = data_directory / '{}.csv'.format(table)
+
+        # If we are loading the geo sample data, handle the data types
+        # specifically so that PostGIS understands them as geometries.
+        if table == 'geo':
```

Comment:
Does it make sense to split this out into its own click function? We can do that in a follow-up.

Reply:
maybe you should check first if the version is equals or more than 3.6